src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -1,15 +1,28 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Truck
- * Do not remove no-field constructor
- */
 public class Truck extends Machine {
+    private float payload;
+
     public Truck() {
     }
 
+    public Truck(String name,String color,float payload) {
+        super();
+        this.setName(name);
+        this.setColor(color);
```

Comment:
it's better to add a constructor to the Machine class and inherit it here like that 
```suggestion
        super(name, color);
```